libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Actions/ActionScope.cs

```diff
@@ -202,7 +202,6 @@ protected virtual async Task<DialogTurnResult> BeginActionAsync(DialogContext dc
             {
                 { "DialogId", action.Id },
                 { "Kind", $"Microsoft.{actionName}" },
-                { "Instance", JsonConvert.SerializeObject(action, new JsonSerializerSettings() { NullValueHandling = NullValueHandling.Ignore }).ToString() }
             };
```

Comment:
>} [](start = 12, length = 1)

We should add "action.Id" which is the active step.

Reply:
I would do this:
            var properties = new Dictionary<string, string>()
            {
                { "DialogId", this.Id },
                { "Kind", $"Microsoft.{actionName}" },
                { "Instance", action.Id }
            };


---
In reply to: [419624685](https://github.com/microsoft/botbuilder-dotnet/pull/3857#discussion_r419624685) [](ancestors = 419624685)